Bottom: e60970fdce203573c39315d8cf13ab5e4df305de
Top:    503b8d85341e58292dcaa7af38191083b414ee5d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-02 14:11:28 +0000

Refresh of move-polygon-clipped-test-out-

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index e737fcf..bf374ee 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -334,6 +334,9 @@ void
 common_thindraw_pcb_polygon (hidGC gc, PolygonType *poly,
                              const BoxType *clip_box)
 {
+  if (poly->Clipped == NULL)
+    return;
+
   thindraw_contour (gc, poly->Clipped->contours);
   PolygonHoles (poly, clip_box, thindraw_hole_cb, gc);
 }
